### PR TITLE
VPN interface

### PR DESCRIPTION
**Pull request by demonethic-office**

### CODE REVIEW

**Comment by PRMerger18** [comment]
@demonethic-office : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

File: hub/apps/design/shell/tiles-and-notifications/firewall-allowlist-config.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ Many enterprises use firewalls to block unwanted network traffic and ports; unfo
 
 ## Proxy Support
 
-> [!Note]
-> WNS Push Notifications *does not* support proxies. For best results, the connection to WNS must be a direct connection.
+> [!Note]V
```
Comment (hickeys) [inline hub/apps/design/shell/tiles-and-notifications/firewall-allowlist-config.md]: Please remove the V at the end of this line.

**Review by hickeys — CHANGES_REQUESTED** [review]
Just one quick change and this is good to go

**Comment by PRMerger16** [comment]
@demonethic-office : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by hickeys — APPROVED** [review]

**Pull request merged**